webpack/assets/javascripts/react_app/components/common/forms/NumericInput.scss

```diff
@@ -0,0 +1,88 @@
+
```

Comment:
You add `@import "~@theforeman/vendor/scss/variables";` to the beginning of the file to have access to pf3/pf4 scss variables that you can use instead hard-coded values.

IMO it makes sense to reuse them for colors, shadows, sizes, and spaces.

Those are all the variables and the mixins available for use:
- https://theforeman-vendor-docs-v4-14-2.surge.sh/scss/variables.scss
- https://theforeman-vendor-docs-v4-14-2.surge.sh/scss/mixins.scss

It might be easier to search using the patternfly docs:
- PF3 - https://www.patternfly.org/v3/styles/color-palette/
- PF4 - https://www.patternfly.org/v4/documentation/overview/global-css-variables

Reply:
I added this import and change the colors and some of the sizes. Let me know if I missed some changes.